<ui version="4.0" >
 <class>ImportDictSelectSourcePage</class>
 <widget class="QWidget" name="ImportDictSelectSourcePage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Simon kann drei verschiedene Typen von Wörterbüchern importieren: HADIFIX-BOMP Wörterbücher, Wiktionary-Dumps und HTK Wörterbücher.

Bitte selektieren Sie welchen Typ von Wörterbuch Sie importieren möchten:</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rbHadifixBOMP" >
     <property name="text" >
      <string>Hadifix-BOMP</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rbWiktionary" >
     <property name="text" >
      <string>Wiktionary</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rbHTK" >
     <property name="text" >
      <string>HTK Lexikon</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rbNone" >
     <property name="text" >
      <string>Keines (Überspringt diesen Schritt)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
